k8s/commander.sh

```diff
@@ -31,6 +32,12 @@ function deploy {
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" rollouts.status-cron
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" deployments.create-web --apply --tag full-${COMMIT_HASH}
     invoke  -f "regions/${REGION}/${REGION_ENV}.yaml" rollouts.status-web
+
+    # run post-deployment tasks
+    echo "Running post-deployment tasks"
+    SUMO_POD=$(${KUBECTL_BIN} -n "${K8S_NAMESPACE}" get pods -o=jsonpath={.items[0].metadata.name})
```

Comment:
[The docs](https://kubernetes.io/docs/reference/kubectl/jsonpath/) have single quotes around the value passed to `jsonpath`, e.g.

`kubectl get pods -o=jsonpath='{.items[0].metadata.name}'`

Are those needed here?

Reply:
I tried running exactly this and it seemed to work. might be better to quote. I'll try it.

Reply:
with quotes works too and seems safer. updated.